data/sql/updates/pending_db_world/rev_1584403279317487100.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584403279317487100');
+
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584283998126323200');
```

Comment:
this line is duplicate, only one should be added

Reply:
i think grimgravy took the wrong one (should take the first one to match the filename)